app/screens/channel_members/channel_members.js

```diff
@@ -239,6 +239,18 @@ export default class ChannelMembers extends PureComponent {
                 enabled={enabled}
             />
         );
+    }
+
+    renderUnselectableItem = (props) => {
```

Comment:
Nit: Looks similar to `renderSelectableItem ` so i think we can use the existing func with adding couple of arguments and call 

```
renderUnselectableItem = () => {
   return this.renderSelectableItem(this.props, false, false); // added params for `selectable` and `enabled `
}

Reply:
@migbot Nice!. Thanks, Yup looks good!